File: Java/AlgoExpert/Sorting/Easy/Selection Sort.java
import java.util.*;

class Program {
  public static int[] selectionSort(int[] array) {
    // Write your code here.
		
		for(int i = 0; i < array.length - 1; i++){
			int minimum = i;
			for(int j = i + 1; j < array.length; j++){
				if(array[minimum] > array[j]){
					minimum = j;
				}
				
			}
			
			int temp = array[i];
				array[i] = array[minimum];
				array[minimum] = temp;
		}
		
    return array;
  }
}

File: Python/AlgoExpert/Sorting/Easy/selectionSort.py
def selectionSort(array):
    # Write your code here.
    for i in range(len(array) - 1):
        smallest_idx = i
        for current_idx in range(i + 1, len(array)):
            if array[current_idx] < array[smallest_idx]:
                smallest_idx = current_idx
        swap(array, i, smallest_idx)
    return array


def swap(array, i, j):
    array[i], array[j] = array[j], array[i]
